From 146f50372a476e6a150d8d86010746617915b39f Mon Sep 17 00:00:00 2001
From: David Daney <david.daney@cavium.com>
Date: Tue, 14 May 2013 14:19:45 -0700
Subject: [PATCH 151/518] MIPS: OCTEON: Reword octeon_pci_console messages.

Source: Cavium Networks, Inc.
MR: 00000
Type: Integration
Disposition: Merged from Octeon Tree
ChangeID: c4edf706afa9a947117d5c9d7265b5e64ae28f1e
Description:

Printing "ERROR:" frightens people, so tone things down a bit.

Signed-off-by: David Daney <david.daney@cavium.com>
Signed-off-by: Leonid Rosenboim <lrosenboim@caviumnetworks.com>
Signed-off-by: Corey Minyard <cminyard@mvista.com>
[Original patch taken from Cavium SDK 3.1.1 525]
Signed-off-by: Yanjiang Jin <yanjiang.jin@windriver.com>
---
 arch/mips/cavium-octeon/octeon-pci-console.c | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/arch/mips/cavium-octeon/octeon-pci-console.c b/arch/mips/cavium-octeon/octeon-pci-console.c
index 0e3f59b..1d4edc3 100644
--- a/arch/mips/cavium-octeon/octeon-pci-console.c
+++ b/arch/mips/cavium-octeon/octeon-pci-console.c
@@ -12,6 +12,9 @@
  * client side of the protocol when the OCTEON SOC is in PCI target
  * mode.
  */
+
+#define pr_fmt(fmt) KBUILD_MODNAME ": " fmt
+
 #include <linux/console.h>
 #include <linux/mutex.h>
 #include <linux/tty.h>
@@ -326,8 +329,10 @@ static __init int octeon_pci_console_module_init(void)
 	if (!octeon_pci_console.ttydrv)
 		return 0;
 	if (octeon_pci_console_setup0(&octeon_pci_console)) {
-		pr_err("ERROR: octeon_pci_console_setup0 failed.\n");
+		pr_notice("Console not created.\n");
 		return -ENODEV;
+	} else {
+		pr_info("Initialized.\n");
 	}
 
 	octeon_pci_console.ttydrv->owner = THIS_MODULE;
-- 
1.9.1

